tools/serving-bridge/tfx_serving_test.go

```diff
@@ -45,6 +45,9 @@ func gatherFiles(dir string) (files []string, err kv.Error) {
 }
 
 func TFXUploadModel(ctx context.Context, s3Client *minio.Client, cfg Config, modelDir string, baseDir string) (indexS3Info minio.ObjectInfo, objsCreated []minio.ObjectInfo, err kv.Error) {
+
+	logger.Debug("stack", stack.Trace().TrimRuntime())
+
 	files, err := gatherFiles(modelDir)
```

Comment:
Will this be left in final code?

Reply:
Yes, part of the test system.  I have it there because I want to see the feature test bed down a little. It'll get removed in the future I think, once I am confident in the test itself.  Tests are much more difficult to stabilize than the actual production code.